apps/remote_control/lib/lexical/remote_control/completion/struct.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Lexical.RemoteControl.Completion.Struct do
```

Comment:
The documentation is complicating the rest of this PR. We don't presently add documentation to other things, and I think we should table this

Reply:
Ok, we can do the documentation part in another pr